clang/test/SemaSYCL/sycl-attr-warn-non-kernel.cpp

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -fsyntax-only -verify -pedantic %s
+
+// The test check issuing diagnostics for attributes that can not be applied to a non SYCL kernel function
+
+#include "sycl.hpp" //clang/test/SemaSYCL/Inputs/sycl.hpp
+
```

Comment:
What is the required behavior for functions marked with SYCL_EXTERNAL? Can you add tests for this as well?

Reply:
Sure I will add.